CHANGELOG.md

```diff
@@ -5,7 +5,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [3.1.1] 2022-02-09
+## [3.1.2] - 2022-02-24
+
+### Added
+
+- `wp_parsely_enable_admin_bar` filter. [#691](https://github.com/Parsely/wp-parsely/pull/691)
+
+### Changed
+
+- Don't return metadata on REST if no API key. [#688](https://github.com/Parsely/wp-parsely/pull/688)
+
+### Fixed
+
+- Compatibility issues when the default category wasn't `Uncategorized`. [#620](https://github.com/Parsely/wp-parsely/pull/620)
+- Fixed some type safety issues. [#657](https://github.com/Parsely/wp-parsely/pull/657)
+- Update outdated references in function comments. [#676](https://github.com/Parsely/wp-parsely/pull/676)
+- Exclude some files & dirs from exported plugin archives. [#612](https://github.com/Parsely/wp-parsely/pull/612)
+- Not installing wp-env globally on CI. [#665](https://github.com/Parsely/wp-parsely/pull/665)
+- README.md: Change "meta data" to "metadata". [#614](https://github.com/Parsely/wp-parsely/pull/614)
+- Update CONTRIBUTING.md for clarity and readability. [#613](https://github.com/Parsely/wp-parsely/pull/613)
+- Adding help for local development issues. [#616](https://github.com/Parsely/wp-parsely/pull/616)
+- Improve documentation. [#654](https://github.com/Parsely/wp-parsely/pull/654)
+- Minor additions to the contributing file. [#683](https://github.com/Parsely/wp-parsely/pull/683)
```

Comment:
I would consolidate all of these down to a single "Improve documentation" entry; we don't need 5 entries for fixing up docs.